mantidproject/mantid
# Made "No RB number" warning more user friendly in Eng Diff GUI

Previously, an irritating popup box appeared in the Engineering Diffraction GUI if no RB number was entered. It wasn't entirely obvious how to get rid of it (enter an RB number) and you couldn't close it. Now, a label in red appears next to the RB number box asking the user for input, in line with Mantid style.

**To test:**
Re-enable the Engineering Diffraction GUI (like [so](https://github.com/mantidproject/mantid/commit/79056b417ef1e5e3af233f2268534d94b7476957)). Build, and then open up the GUI. On startup you should get the red warning appearing (assuming there is no RB number in there). On entering a number (or any text - there is no validation except for non-empty on the input), the message should disappear and the buttons etc on the GUI become enabled. Try going back and forth a few times, and make sure the tooltip is visible.

Fixes #20979 

**Release Notes** 
[Here](https://github.com/mantidproject/mantid/commit/5e72add2d6679efcf33f4bece0d6db04547bd0ce)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
